plugins/modules/k8s_log.py

```diff
@@ -51,7 +51,8 @@
     description:
     - Use to specify the container within a pod to grab the log from.
     - If there is only one container, this will default to that container.
-    - If there is more than one container, this option is required.
+    - If there is more than one container, this option is required or set I(all_containers) to C(true).
+    - Ignored when I(all_containers) is set to C(true).
```

Comment:
Can we just set `container` and `all_containers` to mutually exclusive? This will be a more consistent experience for users.